scripts/brp-remove-la-files

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" ] || [ "$RPM_BUILD_ROOT" = "/" ]; then
+  exit 0
+fi
+
+find "$RPM_BUILD_ROOT" -name "*.la" -delete -print
```

Comment:
You'll want to add `-type f` there, just in case. Traditionally brp-scripts work quietly so the `-print` should probably go.

Reply:
Oh and please explain the rationale in the commit message. The commit itself shows that a file named brp-remove_la_files gets added, but what it does and why is what the commit summary + message should explain.